internal/api/starlark_test.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"awans.org/aft/internal/bus"
+	"awans.org/aft/internal/db"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestCreateAndrewType(t *testing.T) {
```

Comment:
I'd like this test to actually construct the custom datatype inline (or in a helper) rather than relying on having it defined elsewhere..